2019-03-29 10:31:43.246 18469 DEBUG oslo_messaging._drivers.amqpdriver [-] received message with unique_id: 10d2d59a068042e991e0e1514fda68b5 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:257
2019-03-29 10:31:43.250 18469 DEBUG oslo_messaging._drivers.amqpdriver [req-0540b44d-0082-4ce1-89eb-317d598bb388 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CALL msg_id: 3dd3b0a6c84a4522bd52a25b09e22f07 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:31:43.276 18469 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 3dd3b0a6c84a4522bd52a25b09e22f07 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:31:43.276 18469 WARNING nova.image.glance [req-0540b44d-0082-4ce1-89eb-317d598bb388 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] No protocol specified in for api_server 'http://172.30.126.56:9292', please update [glance] api_servers with fully qualified url including scheme (http / https)
2019-03-29 10:31:43.277 18469 DEBUG nova.objects.instance [req-0540b44d-0082-4ce1-89eb-317d598bb388 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] Lazy-loading 'flavor' on Instance uuid f5a19c94-1a45-4ea0-9386-77c10705c825 obj_load_attr /usr/lib/python2.7/site-packages/nova/objects/instance.py:1034
2019-03-29 10:31:43.278 18469 DEBUG oslo_messaging._drivers.amqpdriver [req-0540b44d-0082-4ce1-89eb-317d598bb388 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CALL msg_id: 274f54c42baf4ae2adcc9ba0c1f44cb1 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:31:43.341 18469 DEBUG oslo_messaging._drivers.amqpdriver [-] CALL msg_id: bf332ba3f1ec4c99ad13a928cfee93e5 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:31:43.343 18469 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 274f54c42baf4ae2adcc9ba0c1f44cb1 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:31:43.350 18469 DEBUG nova.objects.instance [req-0540b44d-0082-4ce1-89eb-317d598bb388 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] Lazy-loading 'pci_devices' on Instance uuid f5a19c94-1a45-4ea0-9386-77c10705c825 obj_load_attr /usr/lib/python2.7/site-packages/nova/objects/instance.py:1034
2019-03-29 10:31:43.351 18469 DEBUG oslo_messaging._drivers.amqpdriver [req-0540b44d-0082-4ce1-89eb-317d598bb388 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CALL msg_id: 97892b2818b749a8a8b5edebee1c27fb exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:31:43.381 18469 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: bf332ba3f1ec4c99ad13a928cfee93e5 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:31:43.388 18469 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 97892b2818b749a8a8b5edebee1c27fb __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:31:43.393 18469 DEBUG nova.virt.driver [req-054e3bb9-67aa-419d-9dbf-bf142ec1b6d8 - - - - -] Emitting event <LifecycleEvent: 1553826703.39, f5a19c94-1a45-4ea0-9386-77c10705c825 => Paused> emit_event /usr/lib/python2.7/site-packages/nova/virt/driver.py:1438
2019-03-29 10:31:43.394 18469 INFO nova.compute.manager [req-054e3bb9-67aa-419d-9dbf-bf142ec1b6d8 - - - - -] [instance: f5a19c94-1a45-4ea0-9386-77c10705c825] VM Paused (Lifecycle Event)
2019-03-29 10:31:43.395 18469 DEBUG oslo_messaging._drivers.amqpdriver [req-054e3bb9-67aa-419d-9dbf-bf142ec1b6d8 - - - - -] CALL msg_id: 1cd02e99e05c460a96765569681cc3b6 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:31:43.437 18469 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 1cd02e99e05c460a96765569681cc3b6 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:31:43.440 18469 DEBUG nova.compute.manager [req-054e3bb9-67aa-419d-9dbf-bf142ec1b6d8 - - - - -] [instance: f5a19c94-1a45-4ea0-9386-77c10705c825] Checking state _get_power_state /usr/lib/python2.7/site-packages/nova/compute/manager.py:1176
2019-03-29 10:31:43.443 18469 DEBUG nova.compute.manager [req-054e3bb9-67aa-419d-9dbf-bf142ec1b6d8 - - - - -] [instance: f5a19c94-1a45-4ea0-9386-77c10705c825] Synchronizing instance power state after lifecycle event "Paused"; current vm_state: active, current task_state: suspending, current DB power_state: 1, VM power_state: 3 handle_lifecycle_event /usr/lib/python2.7/site-packages/nova/compute/manager.py:1088
2019-03-29 10:31:43.445 18469 DEBUG oslo_messaging._drivers.amqpdriver [req-054e3bb9-67aa-419d-9dbf-bf142ec1b6d8 - - - - -] CALL msg_id: 2ad1be8a0905440aa5d7d20823ee5b3e exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:31:43.497 18469 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 2ad1be8a0905440aa5d7d20823ee5b3e __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:31:43.498 18469 INFO nova.compute.manager [req-054e3bb9-67aa-419d-9dbf-bf142ec1b6d8 - - - - -] [instance: f5a19c94-1a45-4ea0-9386-77c10705c825] During sync_power_state the instance has a pending task (suspending). Skip.
2019-03-29 10:31:43.700 18469 DEBUG oslo_service.periodic_task [req-7d4ff131-cff3-4e9f-80c4-65e5cb9752cb - - - - -] Running periodic task ComputeManager._poll_rescued_instances run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2019-03-29 10:31:45.295 18469 DEBUG nova.compute.manager [req-0540b44d-0082-4ce1-89eb-317d598bb388 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] [instance: f5a19c94-1a45-4ea0-9386-77c10705c825] Checking state _get_power_state /usr/lib/python2.7/site-packages/nova/compute/manager.py:1176
2019-03-29 10:31:45.300 18469 DEBUG oslo_messaging._drivers.amqpdriver [req-0540b44d-0082-4ce1-89eb-317d598bb388 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CALL msg_id: 8ee453c520694fb98d13bec0ebf858ca exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:31:45.412 18469 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 8ee453c520694fb98d13bec0ebf858ca __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:31:45.414 18469 WARNING nova.image.glance [req-0540b44d-0082-4ce1-89eb-317d598bb388 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] No protocol specified in for api_server 'http://172.30.126.56:9292', please update [glance] api_servers with fully qualified url including scheme (http / https)
2019-03-29 10:31:45.418 18469 DEBUG oslo_messaging._drivers.amqpdriver [req-0540b44d-0082-4ce1-89eb-317d598bb388 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CALL msg_id: 1a0cdba6ab2c4d15a1e12af645ea2ab4 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:31:45.480 18469 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 1a0cdba6ab2c4d15a1e12af645ea2ab4 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407





2019-03-29 10:31:53.344 18469 DEBUG oslo_messaging._drivers.amqpdriver [-] CALL msg_id: eb2708484907479c8f240866d0b51752 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:31:53.399 18469 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: eb2708484907479c8f240866d0b51752 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:31:59.261 18469 DEBUG oslo_messaging._drivers.amqpdriver [-] received message with unique_id: 661b9c7c80a54a5c96d37dc849ab3151 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:257
2019-03-29 10:31:59.267 18469 DEBUG oslo_messaging._drivers.amqpdriver [req-684e33dd-3ee7-4504-ab45-6cf20ee6a377 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CALL msg_id: 60a82268e97f42f89a42388890efb36e exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:31:59.302 18469 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 60a82268e97f42f89a42388890efb36e __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:31:59.302 18469 INFO nova.compute.manager [req-684e33dd-3ee7-4504-ab45-6cf20ee6a377 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] [instance: f5a19c94-1a45-4ea0-9386-77c10705c825] Resuming
2019-03-29 10:31:59.303 18469 WARNING nova.image.glance [req-684e33dd-3ee7-4504-ab45-6cf20ee6a377 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] No protocol specified in for api_server 'http://172.30.126.56:9292', please update [glance] api_servers with fully qualified url including scheme (http / https)
2019-03-29 10:31:59.304 18469 DEBUG nova.objects.instance [req-684e33dd-3ee7-4504-ab45-6cf20ee6a377 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] Lazy-loading 'flavor' on Instance uuid f5a19c94-1a45-4ea0-9386-77c10705c825 obj_load_attr /usr/lib/python2.7/site-packages/nova/objects/instance.py:1034
2019-03-29 10:31:59.306 18469 DEBUG oslo_messaging._drivers.amqpdriver [req-684e33dd-3ee7-4504-ab45-6cf20ee6a377 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CALL msg_id: 603ecba86c65497aa919bbb8ce4179a2 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:31:59.370 18469 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 603ecba86c65497aa919bbb8ce4179a2 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:31:59.376 18469 DEBUG oslo_concurrency.lockutils [req-684e33dd-3ee7-4504-ab45-6cf20ee6a377 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] Acquired semaphore "refresh_cache-f5a19c94-1a45-4ea0-9386-77c10705c825" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2019-03-29 10:31:59.377 18469 DEBUG nova.network.neutronv2.api [req-684e33dd-3ee7-4504-ab45-6cf20ee6a377 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] [instance: f5a19c94-1a45-4ea0-9386-77c10705c825] _get_instance_nw_info() _get_instance_nw_info /usr/lib/python2.7/site-packages/nova/network/neutronv2/api.py:1295
2019-03-29 10:31:59.378 18469 DEBUG oslo_messaging._drivers.amqpdriver [req-684e33dd-3ee7-4504-ab45-6cf20ee6a377 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CALL msg_id: 21ead3a858304ba68121517dfea845a7 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:31:59.399 18469 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 21ead3a858304ba68121517dfea845a7 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:31:59.466 18469 DEBUG neutronclient.v2_0.client [req-684e33dd-3ee7-4504-ab45-6cf20ee6a377 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] GET call to neutron for http://172.30.126.56:9696/v2.0/ports.json?tenant_id=d4bbd972ca1b463ba028ccd1df983917&device_id=f5a19c94-1a45-4ea0-9386-77c10705c825 used request id req-2a2817ca-7e16-49b2-92ec-483cba2a2f2d _append_request_id /usr/lib/python2.7/site-packages/neutronclient/v2_0/client.py:128
2019-03-29 10:31:59.619 18469 DEBUG neutronclient.v2_0.client [req-684e33dd-3ee7-4504-ab45-6cf20ee6a377 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] GET call to neutron for http://172.30.126.56:9696/v2.0/networks.json?id=968d1118-8921-4ef9-9cf6-534ff3d8bf7d used request id req-ee853601-f41a-4f63-a2e9-9f29c10dfa1c _append_request_id /usr/lib/python2.7/site-packages/neutronclient/v2_0/client.py:128
2019-03-29 10:31:59.657 18469 DEBUG neutronclient.v2_0.client [req-684e33dd-3ee7-4504-ab45-6cf20ee6a377 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] GET call to neutron for http://172.30.126.56:9696/v2.0/floatingips.json?fixed_ip_address=10.10.10.11&port_id=b3249515-3aab-439f-ba9c-bc498756871a used request id req-2ccd752a-55a2-4b0e-a115-af6c072d968a _append_request_id /usr/lib/python2.7/site-packages/neutronclient/v2_0/client.py:128
2019-03-29 10:31:59.698 18469 DEBUG oslo_service.periodic_task [req-7d4ff131-cff3-4e9f-80c4-65e5cb9752cb - - - - -] Running periodic task ComputeManager._sync_scheduler_instance_info run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2019-03-29 10:31:59.700 18469 DEBUG oslo_messaging._drivers.amqpdriver [req-7d4ff131-cff3-4e9f-80c4-65e5cb9752cb - - - - -] CALL msg_id: bdcd95387d6b45869209dab9208179b4 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:31:59.746 18469 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: bdcd95387d6b45869209dab9208179b4 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:31:59.749 18469 DEBUG oslo_messaging._drivers.amqpdriver [req-7d4ff131-cff3-4e9f-80c4-65e5cb9752cb - - - - -] CAST unique_id: a4fe38c54ee449098eb6de754aacb7ec FANOUT topic 'scheduler' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:539
2019-03-29 10:31:59.752 18469 DEBUG neutronclient.v2_0.client [req-684e33dd-3ee7-4504-ab45-6cf20ee6a377 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] GET call to neutron for http://172.30.126.56:9696/v2.0/subnets.json?id=b9f5b815-a0ab-4476-a216-6b824fda2957 used request id req-d949a5c4-2d0d-4e44-88ca-280ff2ecac0a _append_request_id /usr/lib/python2.7/site-packages/neutronclient/v2_0/client.py:128
2019-03-29 10:31:59.837 18469 DEBUG neutronclient.v2_0.client [req-684e33dd-3ee7-4504-ab45-6cf20ee6a377 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] GET call to neutron for http://172.30.126.56:9696/v2.0/ports.json?network_id=968d1118-8921-4ef9-9cf6-534ff3d8bf7d&device_owner=network%3Adhcp used request id req-447ccc03-61c4-407e-a7f0-c36ad92a5f3b _append_request_id /usr/lib/python2.7/site-packages/neutronclient/v2_0/client.py:128
2019-03-29 10:31:59.838 18469 DEBUG nova.network.base_api [req-684e33dd-3ee7-4504-ab45-6cf20ee6a377 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] [instance: f5a19c94-1a45-4ea0-9386-77c10705c825] Updating instance_info_cache with network_info: [{"profile": {}, "ovs_interfaceid": "b3249515-3aab-439f-ba9c-bc498756871a", "preserve_on_delete": false, "network": {"bridge": "br-int", "subnets": [{"ips": [{"meta": {}, "version": 4, "type": "fixed", "floating_ips": [], "address": "10.10.10.11"}], "version": 4, "meta": {}, "dns": [], "routes": [], "cidr": "10.10.10.0/24", "gateway": {"meta": {}, "version": 4, "type": "gateway", "address": "10.10.10.1"}}], "meta": {"injected": false, "tenant_id": "d4bbd972ca1b463ba028ccd1df983917", "mtu": 1500}, "id": "968d1118-8921-4ef9-9cf6-534ff3d8bf7d", "label": "net10"}, "devname": "tapb3249515-3a", "vnic_type": "normal", "qbh_params": null, "meta": {}, "details": {"port_filter": true, "ovs_hybrid_plug": true}, "address": "fa:16:3e:29:72:d6", "active": true, "type": "ovs", "id": "b3249515-3aab-439f-ba9c-bc498756871a", "qbg_params": null}] update_instance_cache_with_nw_info /usr/lib/python2.7/site-packages/nova/network/base_api.py:48
2019-03-29 10:31:59.839 18469 DEBUG oslo_messaging._drivers.amqpdriver [req-684e33dd-3ee7-4504-ab45-6cf20ee6a377 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CALL msg_id: e3102c28991847e689a612f653adb2ae exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:31:59.856 18469 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: e3102c28991847e689a612f653adb2ae __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:31:59.857 18469 DEBUG oslo_concurrency.lockutils [req-684e33dd-3ee7-4504-ab45-6cf20ee6a377 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] Releasing semaphore "refresh_cache-f5a19c94-1a45-4ea0-9386-77c10705c825" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2019-03-29 10:31:59.859 18469 DEBUG oslo_messaging._drivers.amqpdriver [req-684e33dd-3ee7-4504-ab45-6cf20ee6a377 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CALL msg_id: 5de33f355d8843618087fc023b39eed2 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:31:59.876 18469 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 5de33f355d8843618087fc023b39eed2 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:31:59.881 18469 DEBUG nova.virt.libvirt.vif [req-684e33dd-3ee7-4504-ab45-6cf20ee6a377 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] vif_type=ovs instance=Instance(access_ip_v4=None,access_ip_v6=None,architecture='x86_64',auto_disk_config=True,availability_zone='nova',cell_name=None,cleaned=True,config_drive='True',created_at=2019-03-29T02:02:33Z,default_ephemeral_device=None,default_swap_device=None,deleted=False,deleted_at=None,device_metadata=<?>,disable_terminate=False,display_description='vm001',display_name='vm001',ec2_ids=<?>,ephemeral_gb=0,ephemeral_key_uuid=None,fault=<?>,flavor=Flavor(1),host='com58',hostname='vm001',id=24,image_ref='',info_cache=InstanceInfoCache,instance_type_id=1,kernel_id='',key_data=None,key_name=None,keypairs=<?>,launch_index=0,launched_at=2019-03-29T02:12:27Z,launched_on='com58',locked=False,locked_by=None,memory_mb=512,metadata={},migration_context=<?>,new_flavor=None,node='com58',numa_topology=<?>,old_flavor=None,os_type=None,pci_devices=<?>,pci_requests=<?>,power_state=4,progress=0,project_id='d4bbd972ca1b463ba028ccd1df983917',ramdisk_id='',reservation_id='r-x56iqcqr',root_device_name='/dev/vda',root_gb=1,security_groups=SecurityGroupList,services=<?>,shutdown_terminate=False,system_metadata={boot_roles='admin',clean_attempts='1',image_architecture='x86_64',image_base_image_ref='',image_min_disk='1',image_min_ram='0',old_vm_state='active'},tags=<?>,task_state='resuming',terminated_at=None,updated_at=2019-03-29T02:31:45Z,user_data=None,user_id='9e9e53917dda421083fa6af425c6bfbf',uuid=f5a19c94-1a45-4ea0-9386-77c10705c825,vcpu_model=<?>,vcpus=1,vm_mode=None,vm_state='suspended') vif={"profile": {}, "ovs_interfaceid": "b3249515-3aab-439f-ba9c-bc498756871a", "preserve_on_delete": false, "network": {"bridge": "br-int", "subnets": [{"ips": [{"meta": {}, "version": 4, "type": "fixed", "floating_ips": [], "address": "10.10.10.11"}], "version": 4, "meta": {}, "dns": [], "routes": [], "cidr": "10.10.10.0/24", "gateway": {"meta": {}, "version": 4, "type": "gateway", "address": "10.10.10.1"}}], "meta": {"injected": false, "tenant_id": "d4bbd972ca1b463ba028ccd1df983917", "mtu": 1500}, "id": "968d1118-8921-4ef9-9cf6-534ff3d8bf7d", "label": "net10"}, "devname": "tapb3249515-3a", "vnic_type": "normal", "qbh_params": null, "meta": {}, "details": {"port_filter": true, "ovs_hybrid_plug": true}, "address": "fa:16:3e:29:72:d6", "active": true, "type": "ovs", "id": "b3249515-3aab-439f-ba9c-bc498756871a", "qbg_params": null} plug /usr/lib/python2.7/site-packages/nova/virt/libvirt/vif.py:776
2019-03-29 10:31:59.882 18469 DEBUG nova.network.os_vif_util [req-684e33dd-3ee7-4504-ab45-6cf20ee6a377 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] Converting VIF {"profile": {}, "ovs_interfaceid": "b3249515-3aab-439f-ba9c-bc498756871a", "preserve_on_delete": false, "network": {"bridge": "br-int", "subnets": [{"ips": [{"meta": {}, "version": 4, "type": "fixed", "floating_ips": [], "address": "10.10.10.11"}], "version": 4, "meta": {}, "dns": [], "routes": [], "cidr": "10.10.10.0/24", "gateway": {"meta": {}, "version": 4, "type": "gateway", "address": "10.10.10.1"}}], "meta": {"injected": false, "tenant_id": "d4bbd972ca1b463ba028ccd1df983917", "mtu": 1500}, "id": "968d1118-8921-4ef9-9cf6-534ff3d8bf7d", "label": "net10"}, "devname": "tapb3249515-3a", "vnic_type": "normal", "qbh_params": null, "meta": {}, "details": {"port_filter": true, "ovs_hybrid_plug": true}, "address": "fa:16:3e:29:72:d6", "active": true, "type": "ovs", "id": "b3249515-3aab-439f-ba9c-bc498756871a", "qbg_params": null} nova_to_osvif_vif /usr/lib/python2.7/site-packages/nova/network/os_vif_util.py:413
2019-03-29 10:31:59.883 18469 DEBUG nova.network.os_vif_util [req-684e33dd-3ee7-4504-ab45-6cf20ee6a377 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] Converted object VIFBridge(active=True,address=fa:16:3e:29:72:d6,bridge_name='qbrb3249515-3a',has_traffic_filtering=True,id=b3249515-3aab-439f-ba9c-bc498756871a,network=Network(968d1118-8921-4ef9-9cf6-534ff3d8bf7d),plugin='ovs',port_profile=VIFPortProfileBase,preserve_on_delete=False,vif_name='tapb3249515-3a') nova_to_osvif_vif /usr/lib/python2.7/site-packages/nova/network/os_vif_util.py:425
2019-03-29 10:31:59.883 18469 DEBUG os_vif [req-684e33dd-3ee7-4504-ab45-6cf20ee6a377 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] Plugging vif VIFBridge(active=True,address=fa:16:3e:29:72:d6,bridge_name='qbrb3249515-3a',has_traffic_filtering=True,id=b3249515-3aab-439f-ba9c-bc498756871a,network=Network(968d1118-8921-4ef9-9cf6-534ff3d8bf7d),plugin='ovs',port_profile=VIFPortProfileBase,preserve_on_delete=False,vif_name='tapb3249515-3a') plug /usr/lib/python2.7/site-packages/os_vif/__init__.py:79
2019-03-29 10:31:59.885 18469 DEBUG oslo.privsep.daemon [-] privsep: request[97287216]: (3, 'vif_plug_ovs.linux_net.ensure_bridge', (u'qbrb3249515-3a',), {}) out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:31:59.885 18469 DEBUG oslo.privsep.daemon [-] privsep: reply[97287216]: (4, None) out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:31:59.887 18469 DEBUG oslo.privsep.daemon [-] privsep: request[97287216]: (3, 'vif_plug_ovs.linux_net.update_veth_pair', ('qvbb3249515-3a', 'qvob3249515-3a', 1500), {}) out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:31:59.888 18469 DEBUG oslo.privsep.daemon [-] Running cmd (subprocess): ip link set qvbb3249515-3a mtu 1500 out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:31:59.899 18469 DEBUG oslo.privsep.daemon [-] CMD "ip link set qvbb3249515-3a mtu 1500" returned: 0 in 0.010s out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:31:59.899 18469 DEBUG oslo.privsep.daemon [-] Running cmd (subprocess): ip link set qvob3249515-3a mtu 1500 out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:31:59.904 18469 DEBUG oslo.privsep.daemon [-] CMD "ip link set qvob3249515-3a mtu 1500" returned: 0 in 0.007s out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:31:59.904 18469 DEBUG oslo.privsep.daemon [-] privsep: reply[97287216]: (4, None) out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:31:59.907 18469 DEBUG oslo.privsep.daemon [-] privsep: request[97287216]: (3, 'vif_plug_ovs.linux_net.update_ovs_vif_port', ('qvob3249515-3a', 1500), {}) out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:31:59.907 18469 DEBUG oslo.privsep.daemon [-] Running cmd (subprocess): ip link set qvob3249515-3a mtu 1500 out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:31:59.911 18469 DEBUG oslo.privsep.daemon [-] CMD "ip link set qvob3249515-3a mtu 1500" returned: 0 in 0.005s out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:31:59.912 18469 DEBUG oslo.privsep.daemon [-] privsep: reply[97287216]: (4, None) out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:31:59.912 18469 INFO os_vif [req-684e33dd-3ee7-4504-ab45-6cf20ee6a377 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] Successfully plugged vif VIFBridge(active=True,address=fa:16:3e:29:72:d6,bridge_name='qbrb3249515-3a',has_traffic_filtering=True,id=b3249515-3aab-439f-ba9c-bc498756871a,network=Network(968d1118-8921-4ef9-9cf6-534ff3d8bf7d),plugin='ovs',port_profile=VIFPortProfileBase,preserve_on_delete=False,vif_name='tapb3249515-3a')
2019-03-29 10:32:00.283 18469 DEBUG nova.virt.libvirt.host [req-054e3bb9-67aa-419d-9dbf-bf142ec1b6d8 - - - - -] Removed pending event for f5a19c94-1a45-4ea0-9386-77c10705c825 due to lifecycle event _event_emit_delayed /usr/lib/python2.7/site-packages/nova/virt/libvirt/host.py:302
2019-03-29 10:32:00.284 18469 DEBUG nova.virt.driver [req-054e3bb9-67aa-419d-9dbf-bf142ec1b6d8 - - - - -] Emitting event <LifecycleEvent: 1553826720.28, f5a19c94-1a45-4ea0-9386-77c10705c825 => Started> emit_event /usr/lib/python2.7/site-packages/nova/virt/driver.py:1438
2019-03-29 10:32:00.284 18469 INFO nova.compute.manager [req-054e3bb9-67aa-419d-9dbf-bf142ec1b6d8 - - - - -] [instance: f5a19c94-1a45-4ea0-9386-77c10705c825] VM Started (Lifecycle Event)
2019-03-29 10:32:00.285 18469 DEBUG oslo_messaging._drivers.amqpdriver [req-054e3bb9-67aa-419d-9dbf-bf142ec1b6d8 - - - - -] CALL msg_id: 2b401d153df444829487ec4de70910fa exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:32:00.297 18469 DEBUG nova.objects.instance [req-684e33dd-3ee7-4504-ab45-6cf20ee6a377 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] Lazy-loading 'pci_devices' on Instance uuid f5a19c94-1a45-4ea0-9386-77c10705c825 obj_load_attr /usr/lib/python2.7/site-packages/nova/objects/instance.py:1034
2019-03-29 10:32:00.299 18469 DEBUG oslo_messaging._drivers.amqpdriver [req-684e33dd-3ee7-4504-ab45-6cf20ee6a377 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CALL msg_id: 13c30d142fb543b2a2ffc1c3102b156c exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:32:00.338 18469 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 13c30d142fb543b2a2ffc1c3102b156c __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:32:00.343 18469 INFO nova.virt.libvirt.driver [-] [instance: f5a19c94-1a45-4ea0-9386-77c10705c825] Instance running successfully.
2019-03-29 10:32:00.344 18469 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 2b401d153df444829487ec4de70910fa __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:32:00.346 18469 DEBUG nova.compute.manager [req-054e3bb9-67aa-419d-9dbf-bf142ec1b6d8 - - - - -] [instance: f5a19c94-1a45-4ea0-9386-77c10705c825] Checking state _get_power_state /usr/lib/python2.7/site-packages/nova/compute/manager.py:1176
2019-03-29 10:32:00.348 18469 DEBUG nova.virt.libvirt.guest [req-684e33dd-3ee7-4504-ab45-6cf20ee6a377 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] [instance: f5a19c94-1a45-4ea0-9386-77c10705c825] Failed to set time: agent not configured sync_guest_time /usr/lib/python2.7/site-packages/nova/virt/libvirt/guest.py:177
2019-03-29 10:32:00.348 18469 DEBUG nova.compute.manager [req-684e33dd-3ee7-4504-ab45-6cf20ee6a377 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] [instance: f5a19c94-1a45-4ea0-9386-77c10705c825] Checking state _get_power_state /usr/lib/python2.7/site-packages/nova/compute/manager.py:1176
2019-03-29 10:32:00.351 18469 DEBUG nova.compute.manager [req-054e3bb9-67aa-419d-9dbf-bf142ec1b6d8 - - - - -] [instance: f5a19c94-1a45-4ea0-9386-77c10705c825] Synchronizing instance power state after lifecycle event "Started"; current vm_state: suspended, current task_state: resuming, current DB power_state: 4, VM power_state: 1 handle_lifecycle_event /usr/lib/python2.7/site-packages/nova/compute/manager.py:1088
2019-03-29 10:32:00.353 18469 DEBUG oslo_messaging._drivers.amqpdriver [req-054e3bb9-67aa-419d-9dbf-bf142ec1b6d8 - - - - -] CALL msg_id: 98d6a2109e1f416ab6b4a4ea50b890b5 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:32:00.357 18469 DEBUG oslo_messaging._drivers.amqpdriver [req-684e33dd-3ee7-4504-ab45-6cf20ee6a377 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CALL msg_id: 9ce2fb10367445409f8c2316012a75cb exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:32:00.418 18469 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 98d6a2109e1f416ab6b4a4ea50b890b5 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:32:00.418 18469 INFO nova.compute.manager [req-054e3bb9-67aa-419d-9dbf-bf142ec1b6d8 - - - - -] [instance: f5a19c94-1a45-4ea0-9386-77c10705c825] During sync_power_state the instance has a pending task (resuming). Skip.
2019-03-29 10:32:00.419 18469 DEBUG nova.virt.driver [req-054e3bb9-67aa-419d-9dbf-bf142ec1b6d8 - - - - -] Emitting event <LifecycleEvent: 1553826720.29, f5a19c94-1a45-4ea0-9386-77c10705c825 => Resumed> emit_event /usr/lib/python2.7/site-packages/nova/virt/driver.py:1438
2019-03-29 10:32:00.419 18469 INFO nova.compute.manager [req-054e3bb9-67aa-419d-9dbf-bf142ec1b6d8 - - - - -] [instance: f5a19c94-1a45-4ea0-9386-77c10705c825] VM Resumed (Lifecycle Event)
2019-03-29 10:32:00.420 18469 DEBUG oslo_messaging._drivers.amqpdriver [req-054e3bb9-67aa-419d-9dbf-bf142ec1b6d8 - - - - -] CALL msg_id: 82205e3428d342f88abd5b89bc37791c exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:32:00.510 18469 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 82205e3428d342f88abd5b89bc37791c __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:32:00.511 18469 DEBUG nova.compute.manager [req-054e3bb9-67aa-419d-9dbf-bf142ec1b6d8 - - - - -] [instance: f5a19c94-1a45-4ea0-9386-77c10705c825] Checking state _get_power_state /usr/lib/python2.7/site-packages/nova/compute/manager.py:1176
2019-03-29 10:32:00.514 18469 DEBUG nova.compute.manager [req-054e3bb9-67aa-419d-9dbf-bf142ec1b6d8 - - - - -] [instance: f5a19c94-1a45-4ea0-9386-77c10705c825] Synchronizing instance power state after lifecycle event "Resumed"; current vm_state: suspended, current task_state: resuming, current DB power_state: 4, VM power_state: 1 handle_lifecycle_event /usr/lib/python2.7/site-packages/nova/compute/manager.py:1088
2019-03-29 10:32:00.516 18469 DEBUG oslo_messaging._drivers.amqpdriver [req-054e3bb9-67aa-419d-9dbf-bf142ec1b6d8 - - - - -] CALL msg_id: 8cd751bfe49a44e496804a6e0528eeda exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:32:00.562 18469 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 9ce2fb10367445409f8c2316012a75cb __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:32:00.565 18469 WARNING nova.image.glance [req-684e33dd-3ee7-4504-ab45-6cf20ee6a377 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] No protocol specified in for api_server 'http://172.30.126.56:9292', please update [glance] api_servers with fully qualified url including scheme (http / https)
2019-03-29 10:32:00.571 18469 DEBUG oslo_messaging._drivers.amqpdriver [req-684e33dd-3ee7-4504-ab45-6cf20ee6a377 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CALL msg_id: 75c93b11bca2414ba877ed4981be22ea exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:32:00.573 18469 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 8cd751bfe49a44e496804a6e0528eeda __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:32:00.609 18469 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 75c93b11bca2414ba877ed4981be22ea __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
